koku/masu/external/accounts/db/cur_accounts_db.py

```diff
@@ -44,30 +62,24 @@ def get_accounts_from_source(self, provider_uuid=None, provider_type=None):
             all_providers = collector.get_provider_uuid_map()
             provider = all_providers.get(str(provider_uuid))
             if provider_uuid and not provider:
-                LOG.info(f"Provider for uuid {provider_uuid} does not exist its likely it has been deleted.")
+                LOG.info(log_json(msg="provider does not exist", provider_uuid=provider_uuid))
                 return []
             elif provider_uuid and provider:
-                if provider.active and not provider.paused:
+                if self.source_is_pollable(provider):
                     return [self.get_account_information(provider)]
-                LOG.info(
-                    f"Provider {provider.uuid} is active={provider.active} "
-                    f"or paused={provider.paused}. Processing suspended..."
-                )
                 return []
 
             for _, provider in all_providers.items():
-                if provider.active is False or provider.paused:
-                    LOG.info(
-                        f"Provider {provider.uuid} is active={provider.active} "
-                        f"or paused={provider.paused}. Processing suspended..."
-                    )
-                    continue
                 if provider_type and provider_type not in provider.type:
                     continue
-                accounts.append(self.get_account_information(provider))
-            msg = f"""Looping through providers polling for:
-                provider_uuid: {provider_uuid},
-                provider_type: {provider_type}
-            """
-            LOG.info(msg)
+                if self.source_is_pollable(provider):
+                    accounts.append(self.get_account_information(provider))
+
+            LOG.info(
+                log_json(
+                    msg="looping through providers polling for accounts",
```

Comment:
So, I mostly did the lowercase, log_json transition to everything in the file. Admittedly I didn't put much thought into the logs themselves because I was focused on the disable source stuff. Taking a second look now, 

I don't think the provider_uuid path ever hits this log message. So its really just the provider type. I can move it before the for loop above and remove the `provider_uuid` from the message.

Reply:
New output:
```
koku-koku-worker-1  | [2023-06-23 19:09:40,029] INFO b1f1bfda-8bc0-4d98-9320-7266445f070f {'message': 'looping through providers polling for accounts', 'tracing_id': '', 'provider_type': 'Azure'}
koku-koku-worker-1  | [2023-06-23 19:09:40,029] INFO b1f1bfda-8bc0-4d98-9320-7266445f070f {'message': 'processing disabled for source', 'tracing_id': '', 'source_uuid': '184262c7-d358-47c8-ab53-91453a491477', 'appName': 'Cost Management', 'environment': 'development'}
koku-koku-worker-1  | [2023-06-23 19:09:40,029] INFO b1f1bfda-8bc0-4d98-9320-7266445f070f Polling for
koku-koku-worker-1  |  schema_name: org1234567
koku-koku-worker-1  |  provider: Azure-local
koku-koku-worker-1  |  account (provider uuid): 35187fa9-d41f-4343-b46e-3dd4dee25e05
koku-koku-worker-1  | [2023-06-23 19:09:40,033] INFO b1f1bfda-8bc0-4d98-9320-7266445f070f {'message': 'getting June 2023 report files', 'tracing_id': UUID('35187fa9-d41f-4343-b46e-3dd4dee25e05'), 'schema': 'org1234567', 'provider_uuid': UUID('35187fa9-d41f-4343-b46e-3dd4dee25e05')}
koku-koku-worker-1  | [2023-06-23 19:09:40,033] INFO b1f1bfda-8bc0-4d98-9320-7266445f070f {'message': 'starting manifest processing', 'tracing_id': UUID('35187fa9-d41f-4343-b46e-3dd4dee25e05'), 'schema': 'org1234567', 'provider_uuid': UUID('35187fa9-d41f-4343-b46e-3dd4dee25e05')}
koku-
```